azure-automation-plus-dsc-lab/nested/17.25.00.createAdsServers.json

```diff
@@ -0,0 +1,143 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adsPrefix": {
+      "type": "string"
+    },
+    "azureRegion": {
+      "type": "string"
+    },
+    "adminUserName": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "adsAvSetId": {
+      "type": "string"
+    },
+    "adsNicIds": {
+      "type": "array"
+    },
+    "saName": {
+      "type": "string"
+    },
+    "domainName": {
+      "type": "string"
+    },
+    "domainJoinOptions": {
+      "type": "string"
+    }
+  },
+  "variables": {
+  "ads01name": "[concat(parameters('adsPrefix'),2)]",
+    "adsVmSize": "Standard_A1_v2",
+    "adsServerInstances": 2,
+    "imagePublisher": "MicrosoftWindowsServer",
+    "imageOffer": "WindowsServer",
+    "imageSku": "2016-Datacenter-Server-Core-smalldisk",
+    "saType": "Standard_LRS",
+    "saUrlSuffix": ".blob.core.windows.net",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#referencing-resource-properties) for how to handle storage account uris